### PR TITLE
BE User profile

### PR DESCRIPTION
**Pull request by deciocandido**

### CODE REVIEW

File: src/main/java/se/kth/sda/skeleton/profile/Profile.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+@Entity
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Profile {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/profile/Profile.java:20]: Could add `@NotBlank `annotation

File: src/main/java/se/kth/sda/skeleton/profile/Profile.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    private String displayDescription;
+
+    @Column(nullable = false)
+    private String displayName;
+
+    @Column(nullable = false)
+    private Boolean activeProfile;
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/profile/Profile.java]: Could also add `@NotBlank` or `@NotNull` annotation for every time you use `@Column(nullable = false)` for extra checking

Comment (deciocandido) [reply src/main/java/se/kth/sda/skeleton/profile/Profile.java]: Added the annotation to the field

**Comment by eeels22** [comment]
Thanks for your work Decio. Closing as the user / profile model has changed

**Pull request closed without merging**